modules/api/src/main/java/org/eclipse/xpanse/api/config/GetCspInfoFromRequest.java

```diff
@@ -57,7 +55,7 @@ public class GetCspInfoFromRequest {
     @Resource
     private DatabaseServiceMigrationStorage serviceMigrationStorage;
     @Resource
-    private DatabaseServiceStateManagementTaskStorage managementTaskStorage;
+    private DatabaseServiceOrderStorage serviceOrderStorage;
     @Resource
     private DatabaseServiceOrderStorage serviceOrderTaskStorage;
```

Comment:
Repeatedly important the same bean of `DatabaseServiceOrderStorage`, please delete one.

Reply:
thank u for reminding this, fixed.